doc/api/packages.md

```diff
@@ -852,20 +852,20 @@ as ES modules and `.cjs` files are always treated as CommonJS.
 added: v12.7.0
 changes:
   - version:
+    - v13.7.0
     - v12.16.0
-    - v13.2.0
-    pr-url: https://github.com/nodejs/node/pull/29978
-    description: Implement conditional exports.
+    pr-url: https://github.com/nodejs/node/pull/31008
+    description: Implement logical conditional exports ordering.
   - version:
-    - v12.15.0
     - v13.7.0
+    - v12.16.0
```

Comment:
```suggestion
    - v12.15.0
```

`v12.16.0` is the correct version, this is being addressed in https://github.com/nodejs/node/pull/35428, but I shouldn't have included the change here.